### ISSUE TITLE
Docker Compose Up - Admin account password not honored by env var

### ISSUE THREAD

**Issue by joao-conde** [PrestaShop/PrestaShop#35140]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The env var to define an admin password is not honored.

### Expected behavior

Able to login with that password.

### Steps to reproduce

1. Run docker compose up
2. Try to login and cant

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.0

### If your bug is related to a module, specify its name and its version

none

### Your company or customer's name goes here (if applicable).

none

**Comment by matks** [comment]
Hello @joao-conde what docker image are you using? We have a lot of them 😄 

https://hub.docker.com/r/prestashop/prestashop/
https://github.com/PrestaShop/prestashop-flashlight
https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml

**Comment by joao-conde** [comment]
Apologies for the not-so-descriptive issue. I was just frustrated.

I am using docker-compose up from the master branch of this repository, so the docker file is the one from the master branch.

**Comment by matks** [comment]
@joao-conde Did you try the admin login & password as provided in the docker-compose file?

https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml#L58-L59

thanks @boubkerbribri for the insight

**Comment by joao-conde** [comment]
> @joao-conde Did you try the admin login & password as provided in the docker-compose file?
> 
> https://github.com/PrestaShop/PrestaShop/blob/develop/docker-compose.yml#L58-L59
> 
> thanks @boubkerbribri for the insight

Yeah, precisely that one. Also tried swapping for other values, didnt work. I ended up editing the source code of prestashop manually where he creates the admin account.

**Comment by boo-code** [comment]
Reproducible on 9.2.x, and the cause explains why swapping the value changed nothing.

`.docker/docker_run_git.sh:118` decides the shop is installed from `app/config/parameters.php` alone. The installer writes that file in its **first** step and creates the administrator six steps later:

```
install-dev/controllers/console/process.php
  :117  processGenerateSettingsFile()   -> writes app/config/parameters.php
  :139  processInstallDatabase()
  :143  processInstallDefaultData()
  :146  processPopulateDatabase()
  :150  processConfigureShop()          -> passes admin_password / admin_email, creates the employee
```

So an install that stops anywhere in between - a MySQL hiccup, a killed container, a timeout - leaves a checkout that has `parameters.php` and no administrator. From then on every `docker compose up` prints "PrestaShop Core already installed...", skips the installer, and still prints `ADMIN_MAIL` / `ADMIN_PASSWD` as the credentials. Changing `ADMIN_PASSWD` and re-running does nothing, which matches "also tried swapping for other values, didn't work". `PS_ERASE_DB` is no escape either - that block sits inside the same skipped branch, so setting it to 1 is a silent no-op on an already-"installed" checkout. Editing the source, as you ended up doing, was the only way out.

The script's own failure branch could not tell you either:

```sh
runuser ... index_cli.php ...
if [ $? -ne 0 ]; then
    echo 'warning: PrestaShop installation failed.'
fi
```

`set -e` is set at `:106`, so a failing installer ended the shell before that test - the warning is unreachable, and a failed install prints nothing at all.

`var/.install.prestashop` already distinguishes the two states: the installer creates it alongside `parameters.php` and `Install::finalize()` removes it (`src/PrestaShopBundle/Install/Install.php:1040`). Gating on it as well is enough, and re-running is safe because the CLI installer defaults to `--db_clear=1`.

Measured by running the real script in a throwaway container with `mysql`, `runuser` and `apache2-foreground` stubbed:

```
                                       base                            with the fix
fresh          (no params, no lock)    installs                        installs
half-installed (params + lock)         "already installed", SKIPPED    installs
installed      (params, no lock)       "already installed"             "already installed"
install fails                          no message at all               error message, exit 1
```

Only the two broken states change. Fix prepared on `9.2.x`.

For anyone hitting this today: delete `app/config/parameters.php` and `var/.install.prestashop`, then `docker compose up` again.

Note the banner itself still prints the env credentials unconditionally, which is misleading on a shop installed earlier with different values - #38084 is already open on those exact lines, so this fix leaves them to it.
